lerna.json

```diff
@@ -3,7 +3,7 @@
         "packages/*"
     ],
     "npmClient": "yarn",
-    "version": "1.1.1",
+    "version": "2.0.0",
```

Comment:
Let's revert the version bump here and release 2.0.0 with this PR https://github.com/awslabs/synchro-charts/pull/123